### DISCUSSION TITLE
conda plumed modules

### DISCUSSION THREAD

**Discussion by omaraek** [deepmodeling/deepmd-kit#280] [Q&A] [answered]
Hi everybody, 
is there a way to enable the modules of plumed in the conda environment? 

Thanks

**Comment by njzjz** [comment]
It has been enabled. You can execute `lmp -h` to confirm.

**Comment by omaraek** [comment]
Plumed is enabled for Lammps but the additional modules of Plumed itself are not. For instance, the  crystallization module of Plumed is not accessible in the conda enviorment.
Thanks for your reply.

**Comment by njzjz** [comment, marked as answer]
You can `git clone` and modify https://github.com/deepmd-kit-recipes/plumed-feedstock/blob/15726abab50ebe0bcff4bd21347ef798ecc64212/recipe/build.sh#L27.
Then, `conda install conda-build`, and execute `conda build . -c deepmodeling` in the `recipe` directory.
After you build it successfully, `conda install plumed --use-local`, or `conda install <the path of the package you build>`.

**Comment by omaraek** [comment]
Thanks for your helpful reply. I successfully installed plumed with its modules in conda's environment.
